cmake/develop_dependencies.cmake

```diff
@@ -37,7 +37,7 @@ FetchContent_Declare( disco
 
 FetchContent_Declare( interpolation
     GIT_REPOSITORY  https://github.com/njoy/interpolation
-    GIT_TAG         origin/master
```

Comment:
Is having all of these various {build type}_dependencies.cmake files working for you? Hard to tell if they are different.